src/main/java/jenkins/plugins/logstash/remoteLogging/LoggingMethodFactory.java

```diff
@@ -1,27 +0,0 @@
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
-package jenkins.plugins.logstash.remoteLogging;
-
-import hudson.Extension;
-import hudson.model.AbstractBuild;
-import hudson.model.Run;
-import jenkins.model.logging.LoggingMethod;
-import jenkins.model.logging.LoggingMethodLocator;
-
-/**
- *
- * @author Oleg Nenashev
- */
-@Extension
-public class LoggingMethodFactory extends LoggingMethodLocator {
```

Comment:
Temporary deleted these since I was using an unpatched core, and they are not needed for the Pipeline integration.